mongo/datadog_checks/mongo/collectors/custom_queries.py

```diff
@@ -101,8 +101,10 @@ def _collect_custom_metrics_for_query(self, client, raw_query):
         cursor = pymongo.command_cursor.CommandCursor(
             pymongo.collection.Collection(db, collection_name), result['cursor'], None
         )
+        empty_result_set = True
 
         for row in cursor:
+            empty_result_set = False
```

Comment:
Can't seem to be able to do much better, the CommandCursor iterator does not have a way to check if the iterator has an item without consuming it.